assets/src/components/v2/dup/takeover_alert.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import { Icon } from "../normal_header";
+import FreeText, { FreeTextType } from "./dup_free_text";
+import NormalHeader from "./normal_header";
+
+const LinkArrow = ({ width, color }: any) => {
```

Comment:
I know only DUPs will use this particular SVG, but is it worth adding to the `bundled_svgs` folder? I feel it will be easier to find in the future if it is in its own folder. Totally up to you though.

Reply:
Currently, since this is only used in one niche situation, I think keeping it here is fine. If and when it gets used elsewhere, we can move it for ease of use